src/components/hooks/use-livechat.js

```diff
@@ -1,14 +1,11 @@
 import React from 'react'
 import { getClientInformation, getDomain, getUTMData, isBrowser } from 'common/utility'
 
-export const useLivechat = () => {
+export const useLivechat = (firstLoadOpen) => {
```

Comment:
```suggestion
export const useLivechat = (load_livechat) => {
```
`snake_case` for arguments. recommendation for a clearer name